app/lib/pre_assembly/bundle.rb

```diff
@@ -21,14 +21,13 @@ class Bundle
              :config_filename,
              :content_exclusion,
              :content_md_creation,
+             :content_structure,
```

Comment:
yay - you made it alphabetical again!

Reply:
does your PR make any of these go away?

Reply:
No, that was done separately.  Momentarily, one of the deleted params was reintroduced via a rebase, but @SaravShah fixed that.